python/GafferSceneUI/RenderPassEditor.py

```diff
@@ -531,22 +528,22 @@ def __disableEdits( self, pathListing ) :
 
 		edits = self.__disablableInspectionTweaks( pathListing )
 
-		with Gaffer.UndoScope( self.scriptNode() ), Gaffer.Context( self.context() ) as context :
-			renderPassPath = self.__pathListing.getPath().copy()
-			for path, inspector in edits :
-				renderPassPath.setFromString( path )
-				context["renderPass"] = renderPassPath.property( "renderPassPath:name" )
-				inspection = inspector.inspect()
-				if inspection is not None and inspection.editable() :
-					source = inspection.source()
+		path = pathListing.getPath().copy()
+		with Gaffer.UndoScope( self.scriptNode() ) :
+			for pathString, inspector in edits :
+				path.setFromString( pathString )
+				with Gaffer.Context( path.inspectionContext() ) :
```

Comment:
Another potential `with path.inspectionContext()`

Reply:
As above, removed the context copies from 3ab240b1 and a341a325.